src/ocean/io/select/client/FileSystemEvent.d

```diff
@@ -153,6 +208,21 @@ class FileSystemEvent : ISelectClient
 
     /***********************************************************************
 
+        Replace the handle delegate
+
+        Params:
+            handler = event handler
+
+    ***********************************************************************/
+
+    public void setHandler ( DirHandler handler )
```

Comment:
Ah I see, so the `name` argument of `DirHandler` would be unused, if called for a normal (i.e. not a directory) monitoring?